index.test.js

```diff
@@ -29,6 +29,21 @@ const MockProps = {
   '--text-@media:dark': 'black',
 }
 
+const MockPropsWithCustomAdaptiveProp = {
+  '--red': '#f00',
+  '--pink': '#ffc0cb',
+  '--h': 200,
+  '--s': '50%',
+  '--l': '50%',
+  '--size-1': '1rem',
+  '--size-2': '2rem',
+  '--fade-in': 'fade-in .5s ease',
+  '--fade-in-@': '@keyframes fade-in {to { opacity: 1 }}',
+  '--dark': '@custom-media --dark (prefers-color-scheme: dark);',
+  '--text': 'white',
+  '--text-dark': 'black',
+}
+
```

Comment:
could probably be reduced to just this yeah? less mock data to sift through to find what's being tested?

```suggestion
const MockPropsWithCustomAdaptiveProp = {
  '--text': 'white',
  '--text-dark': 'black',
}
```